docker/local/Dockerfile.local

```diff
@@ -46,9 +47,17 @@ RUN pip install \
 ARG DGL_VERSION=2.3.0
 ARG DGL_CUDA_VERSION=121
 ARG OGB_VERSION=1.3.6
-ARG TORCH_VERSION=2.3
+ARG TORCH_VERSION=2.3.1
```

Comment:
I think we used 2.3.0

Reply:
You mean in the source images? Yeah we can leave it as is for now and update the source images along with this parameter in another PR.